.github/dependabot.yml

```diff
@@ -15,9 +15,9 @@ updates:
       patterns:
         - "*fontsource*"
   ignore:
-  - dependency-name: materialize-css
+  - dependency-name: @materializecss/materialize
     versions:
-    - "> 0.98.2"
+    - "> 1.2.2"
```

Comment:
Can't you just remove it? I defined it because the project wasn't compatible with the 1.0

Reply:
Are maybe why not jumping straight to the version2? Too much changes?

Reply:
> Can't you just remove it? I defined it because the project wasn't compatible with the 1.0

maybe, but I don't want dependabot to try updating to v2 for now

> Are maybe why not jumping straight to the version2? Too much changes?

yes, like the removal of jQuery 😅  will be the scope for another PR